src/Toast/toast.js

```diff
@@ -0,0 +1,6 @@
+import { toastEmitter } from './EventEmitter';
+
+// eslint-disable-next-line import/prefer-default-export
+export const toast = ({ message, duration, actions }) => {
```

Comment:
I think it would be great to add a little [jsDocs documentation](https://jsdoc.app/) for this function